issue_template.md

```diff
@@ -0,0 +1,19 @@
+### ownCloud Core
+
+```
+Firstly, you should add the "classic" ownCloud Core Issue Template here (so that we know, which Database, ownCloud Version and Configuration you are using ;)
+```
+
+### ownCloud Mail
+
+**Mailserver or Service:** (e.g. Outlook, Yahoo, Gmail, Exchange,...)
+
+**Transport Security - IMAP:** (None, SSL, TLS, STARTTLS) 
+
+**Transport Security - SMTP:** (None, SSL, TLS, STARTTLS)
+
+**Mail Transport:** (SMTP or PHP Mail)
+
+**Number of Accounts:**
+
+**ownCloud Mail Version:** (see ownCloud apps page)
```

Comment:
AFAIK that information should be included in the list of enabled apps (core issue template) – so this might be redundant. However, if people are using nightly builds, they should provide the date of downloading it.